Classes/ActorTraits/Mailhog.php

```diff
@@ -71,6 +73,18 @@ public function mailIsAddressedTo(string $address): void
         $this->checkRecipientAddress($address);
     }
 
+    /**
+     * @Then I should see :subject in the email subject
+     * @Then I should see :subject in the email subject and decodeQP :mimeDecodeFlag
+     * @Then I should see :subject in the email subject and decodeQP :mimeDecodeFlag and charset :charset
```

Comment:
vielleicht... klingt etwas besser in Englisch :D 
```suggestion
     * @Then I should see :subject in the email subject with decodeQP :mimeDecodeFlag
     * @Then I should see :subject in the email subject with decodeQP :mimeDecodeFlag and charset :charset
```